### DISCUSSION TITLE
Worthlist

### DISCUSSION THREAD

**Discussion by Sh3r1ffCZ** [Zrips/CMI#7387] [Feature Requests]
Hello, 
I have a suggestion to add a new command where the seller enters the /sell chest command and when you right click on the chest, it sells the contents of the chest. This is a faster and better way of /sell all
Thanks

**Comment by mrfloris** [comment]
Please post feature suggestions here: https://github.com/Zrips/CMI/discussions

**Comment by Zrips** [comment]
Issue with this would come from the fact that container could belong to someone else, so this needs to work with area protection plugins in quite robust way.
